src/main/webapp/app/exam/manage/exam-management.route.ts

```diff
@@ -108,6 +108,10 @@ export const examManagementRoute: Routes = [
         },
         canActivate: [UserRouteAccessService],
     },
+    {
+        path: ':examId/monitoring',
+        loadChildren: () => import('../monitoring/exam-monitoring.module').then((m) => m.ArtemisExamMonitoringModule),
+    },
```

Comment:
Please add authorities to the new route / the ExamM onitoring Dashboard
It is possible to open the exam monitoring dashboard as a student with the link.
This should work:
```suggestion
    },
    data: {
            authorities: [Authority.INSTRUCTOR, Authority.ADMIN],
            pageTitle: 'artemisApp.examManagement.title'examMonitoring',
            },
       
```